### ISSUE TITLE
yarp devel is not configuring if YCM 0.4.0 is installed in the system

### ISSUE THREAD

**Issue by traversaro** [robotology/yarp#1335]
I have `ycm` installed from homebrew, and the latest YARP devel is not configuring with this error:
~~~
CMake Error at /usr/local/share/YCM/modules/AddInstallRPATHSupport.cmake:137 (file):
  file RELATIVE_PATH must be passed a full path to the directory:
  INSTALL_NAME_DIR
Call Stack (most recent call first):
  cmake/YarpOptions.cmake:359 (add_install_rpath_support)
  CMakeLists.txt:63 (include)
~~~

**Comment by traversaro** [comment]
The problem seems related to https://github.com/robotology/yarp/pull/1322 . 
cc @francesco-romano @drdanz

**Comment by traversaro** [comment]
A possible workaround is to simply uninstall the homebrew ycm : `brew uninstall ycm` .

**Comment by traversaro** [comment]
I suspect bumping https://github.com/robotology/yarp/blob/devel/CMakeLists.txt#L16 to the YCM version that will be synced with YARP will be sufficient to solve the issue.

**Comment by francesco-romano** [comment]
Ok. Let's wait for the bump. 
If needed we can update the YCM formula.

**Comment by diegoferigo** [comment]
This bug affects also linux

**Comment by drdanz** [comment]
A possible workaround is to bump YCM required version in YARP...

**Comment by traversaro** [comment]
@drdanz What is the next version of YCM? I think we should bump to that version.

**Comment by drdanz** [comment]
Fixed in 89573412ff2e787a14aca863417d57e066059782
